portfolio/src/main/webapp/script.js

```diff
@@ -29,4 +29,10 @@ function randomizeImage() {
   // Remove the previous image.
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
+}
+
+async function getRandomQuoteUsingAsyncAwait() {
```

Comment:
actually we don't need to add "UsingAsyncAwait" in the function, cleaner to have just `getRandomQuote()`. but this is optional :)